packages/mobile/src/fees/saga.ts

```diff
@@ -0,0 +1,50 @@
+import { getStableTokenContract } from '@celo/contractkit'
+import { call, put, spawn, takeLeading } from 'redux-saga/effects'
+import { Actions, defaultFeeUpdated, FeeType, UpdateDefaultFeeAction } from 'src/fees/actions'
+import { getInvitationVerificationFee } from 'src/invite/saga'
+import { getSendFee } from 'src/send/saga'
+import { CeloDefaultRecipient } from 'src/send/Send'
+import Logger from 'src/utils/Logger'
+
+const TAG = 'fees/saga'
+
+export function* updateDefaultFeeSaga({ feeType }: UpdateDefaultFeeAction) {
+  Logger.debug(TAG + '@updateDefaultFeeSaga', 'updating for ', feeType)
+
+  // TODO: skip fee update if it was calculated recently
```

Comment:
Yeah I think we should also store a timestamp for each fee type and check that here

Reply:
Yes! Do you mind if I do it in another PR?

Reply:
Another Pr makes sense :)